### ISSUE TITLE
Tables in LibreOffice Write.

### ISSUE THREAD

**Issue by GabrieleBattaglia** [nvaccess/nvda#16818]
Hello, I would like to request the implementation of table accessibility in Libre Office Write. Currently, with NVDA, it is possible to insert a table, but nothing else can be done afterwards: you cannot read it, insert text into it, and possibly not even delete or modify it.

**Comment by vsfoote** [comment]
Known enhancement request, #4133 

with duplicates: #1994, #2620,  #4752 , #7610

**Comment by Adriani90** [comment]
Closing as duplicate of #4133
